app/hooks/useCollapsibleHeader.test.tsx

```diff
@@ -32,7 +32,10 @@ describe('useCollapsibleHeader', () => {
         expect(result.current).toEqual({
             defaultHeight: ViewConstants.DEFAULT_HEADER_HEIGHT,
             scrollPaddingTop: LARGE_HEADER_TITLE_HEIGHT,
-            headerHeight: {value: LARGE_HEADER_TITLE_HEIGHT},
+            headerHeight: {
+                get: expect.any(Function),
+                value: LARGE_HEADER_TITLE_HEIGHT,
+            },
```

Comment:
0/5 another way to solve this was the following:
```suggestion
            headerHeight: expect.objectContaining({
                value: LARGE_HEADER_TITLE_HEIGHT,
            }),
```

Not sure if expecting the get was intentional or not.

Reply:
not expected but included by reanimated, can apply the suggested change if more than 1/5

Reply:
Nope. Completely 0/5 :P